weasyprint/css/validation.py

```diff
@@ -685,6 +685,8 @@ def validate_content_token(base_url, token):
             style = args[-1]
             if style in ('none', 'decimal') or style in counters.STYLES:
                 return (name, args)
+        elif prototype in (('string', ['IDENT']),('string', ['IDENT', 'IDENT'])):
+            return (name, args)
```

Comment:
When there are two arguments, this should check that the second one is one of the supported keywords.